testsuite/pom.xml

```diff
@@ -149,7 +149,9 @@
             <modules>
                 <module>test-utils</module>
                 <module>arquillian-utils</module>
+                <!-- rls tmp disable (start)
```

Comment:
@jamezp Is there some plans about missing unit-tests module?

Reply:
@marekkopecky see https://github.com/resteasy/resteasy/pull/4033

Reply:
Thanks!